From ea8c79077c0deb993ba31f55525a5cf03399487a Mon Sep 17 00:00:00 2001
From: Ghennadi Procopciuc <ghennadi.procopciuc@nxp.com>
Date: Mon, 7 Feb 2022 09:19:21 +0200
Subject: [PATCH 28/51] s32: Remove CONFIG_MACH_TYPE

CONFIG_MACH_TYPE is used for arm32 architectures.

Issue: ALB-8483, ALB-8484
Upstream-Status: Pending 

Signed-off-by: Ghennadi Procopciuc <ghennadi.procopciuc@nxp.com>
Signed-off-by: Zhantao Tang <zhantao.tang@windriver.com>
---
 include/configs/s32-gen1.h | 2 --
 include/configs/s32.h      | 2 --
 2 files changed, 4 deletions(-)

diff --git a/include/configs/s32-gen1.h b/include/configs/s32-gen1.h
index 7e37672662..8505755a0f 100644
--- a/include/configs/s32-gen1.h
+++ b/include/configs/s32-gen1.h
@@ -26,8 +26,6 @@
 /* The configurations of this board depend on the definitions in this file and
 * the ones in the header included at the end, configs/s32.h */
 
-#define CONFIG_MACH_TYPE		4146
-
 #define CONFIG_MP
 
 /* Config CACHE */
diff --git a/include/configs/s32.h b/include/configs/s32.h
index 3ee267a1ea..fc6bc687be 100644
--- a/include/configs/s32.h
+++ b/include/configs/s32.h
@@ -12,8 +12,6 @@
 
 #include <generated/autoconf.h>
 
-#define CONFIG_MACH_TYPE		4146
-
 /* Config CACHE */
 #define CONFIG_CMD_CACHE
 
-- 
2.17.1

